UpdateTemplate.cmd

```diff
@@ -0,0 +1,25 @@
+@ECHO OFF
+PUSHD %~dp0
+
+SETLOCAL
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+:: Check if node exists globally
+WHERE node >NUL
+IF NOT '%ERRORLEVEL%'=='0' (
+    ECHO ERROR: UpdateTemplate.cmd requires node installed globally.
+    GOTO :Exit
+)
+
```

Comment:
But I meet the error `'gulp' is not recognized as an internal or external command,
operable program or batch file.`... Seems need to install gulp globally?